docs/core/run-time-config/garbage-collector.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Garbage collector config settings
+description: Learn about run-time settings for configuring how the garbage collector manages memory.
+ms.date: 11/13/2019
+ms.topic: reference
+---
+# Run-time configuration options for garbage collection
+
+This page contains information about garbage collector (GC) settings that can be changed at run time. If you're trying to achieve peak performance of a running app, consider using these settings. However, the defaults provide optimum performance for most applications in typical situations.
```

Comment:
around the beginning of the doc, we should say that the int values for the env vars are interpreted as hex and for other configs are interpreted as dec. the former often gets people.

Reply:
So for example, to specify a heap hard limit of 80K bytes, the values would be 80000 for the JSON file and 13880 for the environment variable? I assume the "0x" prefix should not be included.